ui/src/pages/Dashboard/NodeInfo/NodeApp.vue

```diff
@@ -196,16 +204,45 @@ export default {
       return this.pagination.perPage * (this.pagination.currentPage - 1);
     },
     total() {
-      this.paginationTotal(this.tableData.length);
-      return this.tableData.length;
+      let result;
+      if (this.searchQuery !== '') {
+        const temp = [];
+        result = this.fuseSearch.search(`=${this.searchQuery}`);
+        for (let i = 0; i < Object.keys(result).length; i += 1) {
+          temp.push(result[i].item);
+        }
+        result = temp;
+      } else {
+        result = this.tableData;
+      }
+      this.paginationTotal(result.length);
+      return result.length;
     },
   },
   mounted() {
     this.isLoading = true;
     axios
       .get('https://stats.runonflux.io/fluxinfo?projection=ip,apps')
       .then((response) => {
-        this.tableData = response.data.data;
+        this.values = response.data.data;
+        for (let i = 0, k = 0; i < this.values.length; i += 1) {
```

Comment:
avoid double nested loops. Use like .filter method on the array.

Reply:
Got it